lib/fsm/fsm.go

```diff
@@ -257,7 +257,7 @@ func (f *FSM) RollbackPhase(ctx context.Context, p Params) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	err = CanRollback(plan, p.PhaseID)
+	err = CanRollback(plan, p.PhaseID, p.DryRun)
```

Comment:
No need to pass `dryRun` into CanRollback since it only makes sense to evaluate in case of an actual rollback:
```go
if !p.DryRun {
	err = CanRollback(plan, p.PhaseID)
	if err != nil {
		if !p.Force {
			return trace.Wrap(err)
		}
		f.WithError(err).Warn("Forcing rollback.")
	}
}
```